docs/root/intro/version_history.rst

```diff
@@ -20,6 +20,9 @@ Version history
 * config: use Envoy cpuset size to set the default number or worker threads if :option:`--cpuset-threads` is enabled.
 * config: added support for :ref:`initial_fetch_timeout <envoy_api_field_core.ConfigSource.initial_fetch_timeout>`. The timeout is disabled by default.
 * cors: added :ref:`filter_enabled & shadow_enabled RuntimeFractionalPercent flags <cors-runtime>` to filter.
+* ext_authz: added support for buffering request body.
+* ext_authz: migrated from V2alpha to V2 and improved docs.
```

Comment:
Should we use lowercase instead? `v2alpha`, `v2`, and the same elsewhere.

Just for consistency.